sonar-db/src/main/java/org/sonar/db/DdlUtils.java

```diff
@@ -62,7 +62,9 @@ private static ScriptRunner newScriptRunner(Connection connection) {
     ScriptRunner scriptRunner = new ScriptRunner(connection);
     scriptRunner.setDelimiter(";");
     scriptRunner.setStopOnError(true);
-    scriptRunner.setLogWriter(new PrintWriter(new NullWriter()));
+    StringBuilderWriter out = new StringBuilderWriter();
+    scriptRunner.setLogWriter(new PrintWriter(out));
+    System.err.println(out.toString());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS106)